manifests/s/Splunk/UniversalForwarder/8.2.5/Splunk.UniversalForwarder.locale.en-US.yaml

```
@@ -0,0 +1,14 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Splunk.UniversalForwarder
+PackageVersion: 8.2.5

```

```suggestion
PackageVersion: 8.2.5.0
```
This must match Control Panel.